lib/ReactTags.js

```diff
@@ -234,7 +240,7 @@ ReactTags.propTypes = {
     PropTypes.func,
     PropTypes.element
   ]),
-  inputAttributes: PropTypes.object
+  inputAttributes: PropTypes.object,
```

Comment:
These last two changes appear to conflict with whatever linting settings upstream is using: https://github.com/clyphub/react-tags/commit/87e8c4babbe5606830288bfc83152e1923979943

Not a show stopper for us if `npm test` is passing, but might matter for upstream

Reply:
turns out `npm test` wasn't passing, but it is now! good catch. 12d687d